tapgarden/custodian.go

```diff
@@ -337,7 +337,13 @@ func (c *Custodian) inspectWalletTx(walletTx *lndclient.Transaction) error {
 			return err
 		}
 
-		if c.cfg.ProofCourier == nil || addr == nil {
+		// TODO(ffranr): This proof courier disabled check should be
+		//  removed. It was implemented because some integration test do
+		//  not setup and use a proof courier.
+		proofCourierDisabled := c.cfg.ProofCourierCfg == nil ||
```

Comment:
The variable name `proofCourierDisabled` isn't 100% accurate, as before this was also a fall through if the output didn't have an address of ours.

Reply:
:+1: changed to `skipProofCourier`.